### DISCUSSION TITLE
Browser extension

### DISCUSSION THREAD

**Discussion by MBoui** [nuxt/nuxt#17940] [Questions]
Hi,

I am looking for a help to making a browser extension with Nuxt3. Is someone can point me in the good direction?

**Comment by mblackritter** [comment]
Might check [Medium.com - How I built a browser extension with Vue](https://javascript.plainenglish.io/i-built-a-browser-extension-with-vue-76779d0a6238).

Didn't read it myself so far, but would assume it's a good starting point and generally similar with Nuxt.

**Comment by Benbinbin** [comment]
any starter template for using Nuxt 3 to develop browser extension

**Reply by SpiritusDeos** [reply]
Try to take a look at this template https://github.com/Jervis2049/vite-plugin-crx-mv3

**Reply by Benbinbin** [reply]
Thanks! Now I use [@crxjs/vite-plugin](https://crxjs.dev/vite-plugin) to develop my browser extension [NutMark](https://github.com/Benbinbin/NutMark)
